examples/README.md

```diff
@@ -0,0 +1,109 @@
+The csp-billing-adapter requires a configuration file, (defaults to `~/.config/csp_billing_adapter/config.yaml`)
+in order to run successfully. The configuration files in in the `examples` directory 
```

Comment:
s/in in/in/

Reply:
fixed in https://github.com/SUSE-Enceladus/csp-billing-adapter/pull/38